packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -41,6 +45,29 @@ function SidebarNavigationScreenWrapper( { children, actions } ) {
 	);
 }
 
+function PageLinkInserter( { rootClientId } ) {
+	const { insertBlock } = useDispatch( blockEditorStore );
+	const { getBlockOrder } = useSelect( blockEditorStore );
+	const insertPageLink = useCallback( () => {
```

Comment:
Nit: Probably no need for `useCallback` here. The `SidebarButton` component isn't memoized and will re-render whenever its parent re-renders. So we don't need to pass a referentially stable callback to it.